### PR TITLE
Feat/pan eu validations

### LINKED ISSUE #2008: PanEuropean: data validations

### PR DESCRIPTION
**Pull request by persas**
Closes #2008

### CODE REVIEW

File: src/test/migrations/steps/template.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { BaseProtocol } from '@server/db'
-
-export default async (client: BaseProtocol) => {
-  // DB Step
-}
+// import { BaseProtocol } from '@server/db'
+//
```
Comment (sorja) [inline src/test/migrations/steps/template.ts]: this file doesnt need to be changed

Comment (persas) [reply src/test/migrations/steps/template.ts]: Pre-commit hooks were failing because of "never used". That's why I commented out everything.

Comment (sorja) [reply src/test/migrations/steps/template.ts]: In merge commits you can use --no-verify
👍

File: src/test/migrations/index.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ describe(`Migrations:`, () => {
         .filter((migration: string) => !ranMigrations.includes(migration))
         .sort()
         .map(async (file: string) => {
+          // eslint-disable-next-line no-console
```
Comment (minotogna) [inline src/test/migrations/index.ts]: Can we use Logger here instead of console.log ?

File: tools/dataMigration/panEuropean.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ export const migrate = async (props: {
   console.log('========== START ', new Date().getTime())
   const { assessmentName, legacySchemaName, assessmentLegacy, cycleNames, spec } = props
 
-  console.log('========== 1. delete old assessment')
+  // console.log('========== 1. delete old assessment')
```
Comment (minotogna) [inline tools/dataMigration/panEuropean.ts]: Can we leave console.log active in this file and use ignore comment ?

**Review by sorja — APPROVED** [review]

File: src/test/migrations/index.ts (hunk 2 above)
Comment (sorja) [reply src/test/migrations/index.ts]: Does logger work in this context? Can you check?

**Review by minotogna — APPROVED** [review]

**Pull request merged**